plugin/trino-hudi/src/test/java/io/trino/plugin/hudi/TestHudiSmokeTest.java

```diff
@@ -119,6 +119,18 @@ public void testShowCreateTable()
                         ")");
     }
 
+    @Test
+    public void testMetaColumns()
+    {
+        assertQuery("SELECT _hoodie_commit_time FROM hudi_cow_pt_tbl", "VALUES ('20220906063435640'), ('20220906063456550')");
+        assertQuery("SELECT _hoodie_commit_seqno FROM hudi_cow_pt_tbl", "VALUES ('20220906063435640_0_0'), ('20220906063456550_0_0')");
+        assertQuery("SELECT _hoodie_record_key FROM hudi_cow_pt_tbl", "VALUES ('id:1'), ('id:2')");
+        assertQuery("SELECT _hoodie_partition_path FROM hudi_cow_pt_tbl", "VALUES ('dt=2021-12-09/hh=10'), ('dt=2021-12-09/hh=11')");
+        assertQuery(
+                "SELECT _hoodie_file_name FROM hudi_cow_pt_tbl",
```

Comment:
this should be `$path` for consistency with other data lake connectors. We don't want Hudi to be perceived as different or worse.

do we have an issue for that?

Reply:
As you may already know, `_hoodie_*` columns are stored in a metastore, not generated by the connector on the fly. Also, there's `hudi.columns-to-hide` configuration property for hiding such columns.